user_guide_src/source/incoming/message.rst

```diff
@@ -2,7 +2,7 @@
 HTTP Messages
 =============
 
-The Message class provides an interface to the portions of an HTTP message that are common to both
+The Message class provides an interface to the portions of an HTTP message that is common to both
```

Comment:
"are" is correct. The subject of the relevant sentence fragment: "portions of an HTTP message that **are** ..."